src/dymoprint/labeler.py

```diff
@@ -145,10 +145,10 @@ def tapeColor(self, value):
     def bytesPerLine(self, value: int):
         """Set the number of bytes sent in the following lines. (MLF)"""
 
-        if value < 0 or value + self.dotTab_ > self.max_bytes_per_line(
-            self.tape_size_mm
-        ):
-            raise ValueError
+        #if value < 0 or value + self.dotTab_ > self.max_bytes_per_line(
+        #    self.tape_size_mm
+        #):
+        #    raise ValueError
```

Comment:
You can just delete this.

Interesting how `tape_size_mm` isn't actually doing anything. It's only used in `chainMark`, whatever that means. Let's leave that part in, just so `chainMark` isn't explicitly broken.

```suggestion
        self.tape_size_mm
```